docs/architecture/adr-069-gov-improvements.md

```diff
@@ -0,0 +1,227 @@
+
+```protobuf
+// optimistic_authorized_addreses is an optional governance parameter that limits the authorized accounts than can submit optimisitc proposals
+repeated string optimistic_authorized_addreses = 17 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+
+// Optimistic rejected threshold defines at which percentage of NO votes, the optimistic proposal should fail and be converted to a standard proposal.
+string optimistic_rejected_threshold = 18 [(cosmos_proto.scalar) = "cosmos.Dec"];
+```
+
+#### Multiple Choice Proposal
+
+A multiple choice proposal is a proposal where the voting options can be defined by the proposer.
+
+The number of voting option will be limited to a maximum of 4.
+A new vote option `SPAM` will be added and distinguished of those voting options. `SPAM` will be used to mark a proposal as spam and is explained further below.
+
+Multiple choice proposals, contrary to any other proposal type, cannot have messages to execute. They are only text proposals.
+
+Submitting a new multiple choice proposal will use a different message than the [`v1.MsgSubmitProposal`][5]. This is done in order to simplify the proposal submittion and allow defining the voting options directly.
+
+
+```protobuf
+message MsgSubmitMultipleChoiceProposal {
+  repeated cosmos.base.v1beta1.Coin initial_deposit = 1
+  string proposer = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  string metadata = 3;
+  string title = 4;
+  string summary = 5;
+  string option_one = 6;
+  string option_two = 7;
+  string option_three = 8;
+  string option_four = 9;
+}
+```
+
+Voters can only vote on the defined options in the proposal.
+
+To maintain compatibility with the existing endpoints, the voting options will not be stored in the proposal itself and each option will be mapped to [`v1.VoteOption`][5]. A multiple choice proposal will be stored as a [`v1.Proposal`][5]. A query will be available for multiple choice proposal types to get the voting options.
+
+### Votes
+
+As mentioned above [multiple choice proposal](#multiple-choice-proposal) will introduce an additional vote option: `SPAM`.
+
+This vote option will be supported by all proposal types.
+At the end of the voting period, if a proposal is voted as `SPAM`, it fails and its deposit is burned.
+
+`SPAM` differs from the `No with Veto` vote as its threshold is dynamic.
+A proposal is marked as `SPAM` when the total of weighted votes for all options is lower than the amount of weighted vote on `SPAM`
+(`spam` > `option_one + option_two + option_three + option_four` = proposal marked as spam).
+
+To avoid voters wrongfully voting down a proposal as `SPAM`, voters will be slashed `x`% (default 0%) of their voting stake if they voted `SPAM` on a proposal that wasn't a spam proposal. The parameter allows to incentivise voters to only vote `SPAM` on actual spam proposals and not use `SPAM` as a way to vote `No with Veto` with a different threshold.
```

Comment:
Right but NWV should be 33% or voting power not of weighted votes.

If the quorum is 10% and a proposal just reached quorum, 50% is 5% of VP.

Reply:
Wait, is this true?
https://github.com/cosmos/cosmos-sdk/blob/main/x/gov/keeper/tally.go#L147

Isn't `totalVotingPower` the total of VP that votes? Used also to compute quorum against `totalBonded` which is the total staked VP: https://github.com/cosmos/cosmos-sdk/blob/main/x/gov/keeper/tally.go#L134

So in your example, in a regular prop with 10% quorum and that just reached quorum, you need 33% of 10% to have the prop vetoed so 3.3% VP.

Please correct me if I am wrong.

Reply:
Hmm, well 33% of total bonded would be 7% less than 40% which is the current quorum. I think it's unreasonable to have NWV be that much.
I think minority protection is important, and NWV shouldn't just be used to punish spam proposals.
So the way it's implemented rn is fine for me (33% of total VP that actually votes). 

I do think though NWV now is broken in any case. It's just a freely available oversized NO, with no associated cost.
It should be equally slashed when used and the proposal is not veoted IMHO. Right now there is no cost to voting NWV instead of NO, except the potential social backlash/pressure.

Reply:
For sure 33% would be unfair if the participation is low. We could have a parameter that let chains define the veto threshold as a percentage of the bonded token, with a maximum of 33%. Anyway, let's keep this discussion for a potential RFC.

Reply:
I definitely don't want to drag the discussion on NWV here, it's out of scope.
[But I want to note that even the documentation of `x/gov` is consistent with the code, counting NWV against the total votes and not the total bonded https://github.com/cosmos/cosmos-sdk/blame/main/x/gov/README.md#L473]

I still think though that having a 50% threshold for SPAM and slashing if this threshold is not reached is too harsh without an explicit and always present NO option, and a consequently lower threshold for SPAM.
I am not completely against doing it the way you propose now, but I have some slight doubts so I wanted to share them.

Reply:
The idea is that if there is a big amount of other votes, it is most probably not a spam proposal, so it makes sense that the more votes there is the harder it is to mark a proposal as spam.

Reply:
Yep, and that's why it sorta makes sense to me even the way you propose now. I mean the true spam proposals usually have super high percentages of NWV, well above the veto threshold, this being true on any chain I know of. I would expect that SPAM votes would be much higher than anything else for clear spam.

Tbh I am probably overthinking this part, well for once because I was comparing with NWV on the regular props, and secondly because of the absence of an explicit way to express disagreement with all of the proposed options  (assuming failure of the off-chain vote-option discovery), which is in fact a different aspect.